.github/CODEOWNERS

```diff
@@ -1,3 +1,16 @@
+################
+# Misc
+################
+/*                               @azure/azure-sdk-eng
+/.azure-pipelines/               @azure/azure-sdk-eng
+/.devcontainer/                  @azure/azure-sdk-eng
+/artifacts/                      @azure/azure-sdk-eng
+/config/                         @azure/azure-sdk-eng
+/doc/                            @azure/azure-sdk-eng
+/scripts/                        @azure/azure-sdk-eng
+/src/                            @azure/azure-sdk-eng
+/web/                            @azure/azure-sdk-eng
```

Comment:
Some things like these (src/web) we also eventually want to reorganize under tools directory.

Reply:
Created:
- #5056